package.json

```diff
@@ -58,7 +58,6 @@
     "pouchdb-vhost": "^1.0.2",
     "pouchdb-wrappers": "^1.3.6",
     "raw-body": "^2.2.0",
-    "sanitize-filename": "^1.6.1",
```

Comment:
One problem with removing `sanitize-filename` entirely is that it was doing some extra work for us to ensure that e.g. files named `PRN` or `CON` don't break on Windows (see [this page](https://answers.microsoft.com/en-us/windows/forum/windows_7-files/why-cant-i-create-a-folder-with-name/23c86662-4988-4c7d-9c2d-3e33d4413de3); you really cannot create files or directories with those names)

This might be considered a minor thing, but if you run pouchdb-server on Windows and users can crash your system by creating DBs named `PRN` then it's kind of a big deal 😅